### PR TITLE
feat(rebranding): changed the colors and aplyed the rebranding variables

### PR DESCRIPTION
**Pull request by WillianLomeu**

### CODE REVIEW

File: src/components/UnsuportedContent.vue [hunk 1]
```diff
@@ -65,7 +57,7 @@ export default {
       default: 'Falha ao enviar a mensagem'
```
Comment (EdsonGalindoTakeBlip) [inline src/components/UnsuportedContent.vue:57]: Is this text right? Or is this only a default value that is replaced by the text on the Blip user Language?

Comment (EdsonGalindoTakeBlip) [inline src/components/UnsuportedContent.vue:57]: Is this text right? Or is this only a default value that is replaced by the text on the Blip user Language?

File: src/components/MediaLink/BlipFile.vue [hunk 2]
```diff
@@ -121,18 +121,35 @@ export default {
...
+      .file-icon-wrapper {
+        background-color: $color-surface-2;
+        padding-right: 20px !important;
+        border-radius: 13px 0 0 2px;
+      }
+      .description-wrapper {
+        background-color: $color-surface-1;
+        color: $color-content-default;
+        padding-left: 20px !important;
+        border-radius: 0 13px 13px 0;
```
Comment (VMarchena) [inline src/components/MediaLink/BlipFile.vue:152]: Should the `background color` be different between left and right?

File: src/styles/common.scss [hunk 3]
```diff
@@ -80,15 +80,14 @@
 
   .bubble.right {
       border-radius: 13px 2px 13px 13px;
-      box-shadow: 0 -1px 12px 0 rgba(0, 0, 0, 0.1);
-      background-color: $vue-light-blip;
-      color: $vue-white;
+      background-color: $color-surface-3;
```
Comment (VMarchena) [inline src/styles/common.scss:83]: Should the `background color` be different between `.bubble.right` and `.bubble.left`?

**Review by EdsonGalindoTakeBlip — CHANGES_REQUESTED** [review]

**Review by SylvesterH13 — APPROVED** [review]

File: src/components/MediaLink/BlipFile.vue (hunk 2 above)
Comment (WillianLomeu) [reply src/components/MediaLink/BlipFile.vue:152]: Yes it should. It was a definition of the design team.

File: src/styles/common.scss (hunk 3 above)
Comment (WillianLomeu) [reply src/styles/common.scss:83]: Yes it should. It was a definition of the design team.

File: src/components/UnsuportedContent.vue (hunk 1 above)
Comment (WillianLomeu) [reply src/components/UnsuportedContent.vue:57]: I don't now how this value is defined. The change is only about de colors.

**Review by kelvinjener — APPROVED** [review]

**Review by EdsonGalindoTakeBlip — APPROVED** [review]

**Review by EdsonGalindoTakeBlip — APPROVED** [review]

**Comment by EdsonGalindoTakeBlip** [comment]
Approved

**Pull request merged**